### PR TITLE
Modified FizzBuzz.js

### PR DESCRIPTION
**Pull request by Saylee119**

### CODE REVIEW

File: hw1/01-fizzbuzz.js [hunk 1]
```diff
@@ -8,9 +8,21 @@ Write a program that writes all the numbers from 1 to 100, with some exceptions:
 Use console.log() to write the proper output to the command line.
 
 **/
-
 const fizzbuzz = () => {
-  // Add your code here
+  for (var i = 1; i < 101; i++) {
+    if (i % 3 == 0 && i % 5 == 0){
+      console.log("FizzBuzz");
```
Comment (juhiaugustine) [inline hw1/01-fizzbuzz.js]: F̵i̵z̵z̵B̵u̵z̵z̵ ---> fizzbuzz

File: hw1/01-fizzbuzz.js [hunk 2]
```diff
@@ -8,9 +8,21 @@ Write a program that writes all the numbers from 1 to 100, with some exceptions:
...
+    } 
+    else if (i % 3 == 0){ 
+      console.log("Fizz");
```
Comment (juhiaugustine) [inline hw1/01-fizzbuzz.js]: F̵i̵z̵z̵ ---> fizz

File: hw1/01-fizzbuzz.js [hunk 3]
```diff
@@ -8,9 +8,21 @@ Write a program that writes all the numbers from 1 to 100, with some exceptions:
...
+    }
+    else if (i % 5 == 0){
+      console.log("Buzz");
```
Comment (juhiaugustine) [inline hw1/01-fizzbuzz.js]: B̵u̵z̵z̵  --->  buzz

**Review by juhiaugustine — CHANGES_REQUESTED** [review]
Code looks good to me. Just the output string needs to be in lowercase.

**Review by juhiaugustine — APPROVED** [review]
Your code seems to be neat and good.

**Review by smah29 — APPROVED** [review]

**Review by CzerPDX — APPROVED** [review]
Code looks good! I like how you used if else here. My solution is much less efficient, this is very nice!

**Pull request merged**